script.js

```diff
@@ -938,3 +938,621 @@ if ($(window).width() <= 700) {
 $(function(){
    $.ui.dialog.prototype._focusTabbable = function(){};
 });
+
```

Comment:
good point for that.
Maybe be difficult atm.
Ticket, task, followups and kb concerned.

I think it should be a more global refactor for all js library.

BTW, i could separate fileupload script in another file (but still load everywhere)